manifests/n/Nvidia/CUDA/12.3.2/Nvidia.CUDA.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Nvidia.CUDA
+PackageVersion: 12.3.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
+Installers:
+- Architecture: x64
+  InstallerUrl: https://developer.download.nvidia.com/compute/cuda/12.3.2/local_installers/cuda_12.3.2_546.12_windows.exe
+  InstallerSha256: 3A4B476E1CD70EAABE42881B480F48557481D1EDE0AB5F33B2AF11B8903425B8
+AppsAndFeaturesEntries:
+  - DisplayVersion: "12.3"
```

```suggestion
  - DisplayVersion: "12.3.2"
```
I'm not sure, but I think this might need to match the `PackageVersion` above.